src/Connection/StreamConnection.php

```diff
@@ -313,11 +313,15 @@ public function read()
                 return $data;
 
             case 'bulkString':
```

Comment:
const string comment from Resp3Strategy applies here. Then, you can case on the const

Reply:
Done!